pkg/kubectl/cmd/cmd.go

```diff
@@ -195,9 +195,6 @@ Available Sub-commands:{{range .Commands}}{{if .IsAvailableCommand}}
 Flags:
 {{.LocalFlags.FlagUsages | trimRightSpace}}{{end}}{{ if .HasInheritedFlags}}
 
-Global Flags:
-{{.InheritedFlags.FlagUsages | trimRightSpace}}{{end}}{{if .HasHelpSubCommands}}
```

Comment:
We do not need "Global Flags" section at all, so it doesn't matter HasHelpSubCommands is True or False. Even if it is false, we will see redundant "Global Flags:" string(L198).